src/System.Net.Http/tests/FunctionalTests/CancellationTest.cs

```diff
@@ -2,161 +2,442 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
 using System.Diagnostics;
+using System.Linq;
 using System.IO;
 using System.Net.Test.Common;
 using System.Threading;
 using System.Threading.Tasks;
-
 using Xunit;
-using Xunit.Abstractions;
 
 namespace System.Net.Http.Functional.Tests
 {
-    public class CancellationTest : HttpClientTestBase
+    public class HttpClientHandler_CancellationTest : HttpClientTestBase
     {
-        private readonly ITestOutputHelper _output;
+        [OuterLoop]
+        [Theory]
+        [MemberData(nameof(TwoBoolsAndCancellationMode))]
+        public async Task PostAsync_CancelDuringRequestContentSend_TaskCanceledQuickly(bool chunkedTransfer, bool connectionClose, CancellationMode mode)
+        {
+            if (IsWinHttpHandler || IsNetfxHandler)
+            {
+                // hangs / doesn't cancel
```

Comment:
Is there a new issue number for the hang for WinHttpHandler?

Reply:
Not currently.  Per "I will open issues for the non-netfx issues and include the corresponding numbers in comments before merging", I'll be opening an issue and adding the issue number before submitting.  I took a quick look at the source to see if it was obvious how to address it, but it wasn't immediately clear to me.